frontend/src/lib/components/Orgs/OrgMemberTable.svelte

```diff
@@ -0,0 +1,148 @@
+<script lang="ts">
+  import { Button } from '$lib/forms';
+  import t from '$lib/i18n';
+  import { Icon } from '$lib/icons';
+  import { createEventDispatcher } from 'svelte';
+  import Dropdown from '../Dropdown.svelte';
+  import type { OrgRole, User } from '$lib/gql/types';
+  import OrgRoleText from './OrgRoleText.svelte';
+  import IconButton from '../IconButton.svelte';
+  import AddOrgMemberModal from './AddOrgMemberModal.svelte';
+  import OrgRoleSelect from '$lib/forms/OrgRoleSelect.svelte';
+
+  type TableUser = Pick<User, 'name' | 'locked' | 'username' | 'email' | 'emailVerified'>;
+
+  type Member = {
+    id: string
+    user: TableUser
+    role: OrgRole
+  };
+
+  export let orgId: string;
+
+  export let shownUsers: Member[];
+  export let canManage = false;
+
+  let editingUser: TableUser|undefined = undefined;
+
+  const dispatch = createEventDispatcher();
```

Comment:
Please make the dispatch type-save.
Your `removeMember` handler is actually broken right now.
Type-safety protects us from that sort of thing.

Reply:
Done in commit c6c6d7ec. Thanks for the tip; I'd forgotten that createEventDispatcher can be made type-safe. (I'd seen it elsewhere in our code but forgot about it while I was deep in the org page code).